src/screens/CostsEntryScreen/CostsEntryScreen.tsx

```diff
@@ -1,15 +1,162 @@
 import React from 'react';
-import { Text, View } from 'react-native';
+import { ScrollView } from 'react-native';
 
+import { Formik } from 'formik';
+import { Button } from 'react-native-elements';
+import * as Yup from 'yup';
+
+import { CostElement } from '~/calculation/CostElement';
+import { DatePickerField } from '~/components/form/DatePickerField/DatePickerField';
+import { SegmentRadioField } from '~/components/form/SegmentRadioField/SegmentRadioField';
+import { SelectField } from '~/components/form/SelectField/SelectField';
+import TextField from '~/components/form/TextField/TextField';
```

Comment:
By the way, what's the reason behind longer paths for components (`/DatePickerField/DatePickerField`)? :smile: I would use `index.js` files for each of them.

Reply:
Yes, actually this might be a good idea :+1:

Reply:
Ok, I will add another task for it to do such thing for all files, doing it in another PR might be good idea :smile: what you think?

Reply:
Yes, it will make sense! Let me know if it's added.

Reply:
Done! :D